comptests/s3/test_rest_api_rgw.py

```diff
@@ -150,24 +218,27 @@ def test_user_create_36633(self):
         user_name = f"{self.user_name_prefix}{str(time.perf_counter_ns()).replace('.', '_')}"
         user_params = {
             'display-name': user_name,
-            'uid' : user_name
-         }
-        self.log.info("Step 1: Creating a new IAM user with name %s", str(user_name))
+            'uid': user_name
+        }
+        self.log.info(
+            "Step 1: Creating a new IAM user with name %s", str(user_name))
         loop = asyncio.get_event_loop()
         status, user_info = loop.run_until_complete(self.obj.create_user(user_params))
-        assert status == HTTPStatus.OK , "Not able to create user. Test Failed"
-        self.log.info("Created user details: %s",user_info)
-        self.log.info("Step 2: Verifying that new IAM user is created successfully")
+        assert status == HTTPStatus.OK, "Not able to create user. Test Failed"
+        self.log.info("Created user details: %s", user_info)
+        self.log.info(
+            "Step 2: Verifying that new IAM user is created successfully")
         status, user_info = loop.run_until_complete(self.obj.get_user_info(user_params))
-        assert status == HTTPStatus.OK , "Not able to Get user Info. Test Failed"
-        self.log.info("Get user info output: %s",user_info)
+        assert status == HTTPStatus.OK, "Not able to Get user Info. Test Failed"
+        self.log.info("Get user info output: %s", user_info)
         self.created_users.append(user_params)
-        self.log.info("Step 3: Creating another IAM user with same name %s", str(user_name))
+        self.log.info(
+            "Step 3: Creating another IAM user with same name %s", str(user_name))
         loop = asyncio.get_event_loop()
         status, user_info = loop.run_until_complete(self.obj.create_user(user_params))
-        assert status == HTTPStatus.CONFLICT , "Didn't get the expected error"
-        self.log.info("Create user output: %s",user_info)
-        self.log.info("END : %s",test_case_name)
+        assert status == HTTPStatus.CONFLICT, "Didn't get the expected error"
```

Comment:
If you are keen on verifying the response headers along with error code in the test. It could have some right error message.